src/RPC/middleware.ts

```diff
@@ -8,6 +8,7 @@ import type {
 import { TransformStream } from 'stream/web';
```

Comment:
We need to figure out how to conditionally import this later because in react-native it's in the global scope.

Reply:
Should I make an new issue for this?

Reply:
No, this is already in TypeScript-Demo-Lib.